lib/index.js

```diff
@@ -88,4 +92,12 @@ module.exports = class Rooftop {
     delete post.content.advanced
     return post
   }
+
+  _writeTemplates (compiler, template) {
+    const path = template.path
+    const output = template.output
+
+    if (!template.path) { throw 'missing template.path' } // eslint-disable-line
+    if (!template.output) { throw 'missing template.output' } // eslint-disable-line
```

Comment:
@michaelcd for this type of validation, we're using [joi](#). you can see how we use it here: https://github.com/static-dev/spike-rooftop/blob/master/lib/index.js#L64-L75. you'll want to do your validation in here